README.md

```diff
@@ -1 +1,128 @@
-# bookstore-react
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+  <h3><b>My Bookstore</b></h3>
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[BookStore\] ](#-BookStore-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Future Features ](#future-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+
+# 📖 [BookStore] <a name="about-project"></a>
+
+BookStore
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+1. Linters
+2. CSS
+3. ReactJS
+
+
+### Key Features <a name="key-features"></a>
+- Linters error free
+- SPA
+- Add Book
+- Remove Book
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](Coming soon)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### setup
+<p>Clone this repository to your desired folder</p>
+<a href ="https://github.com/Lawmsangi/bookstore-react.git"></a>
+
+### Install
+
+Install this project with: -Git clone on your cd Desktop using my repository link above
+
+### Usage
+
+-To run the project, copy its path and run it on your online server.
+
+### Run tests
+
+To run tests, run the following command:
+
+npx eslint "**/*.{js,jsx}"
+npx stylelint "*/.{css,scss}"
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Lawmsangi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lawmsangi-varte-baa429244/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+ ## 🔭 Future Features <a name="future-features"></a>
+
+- Test the app using RTL
+- Add animations style
+- Add details page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```